### PR TITLE
[djangosf] Fix test_notifications

### PR DESCRIPTION
**Pull request by mfraezz**

### CODE REVIEW

File: tests/test_notifications.py [hunk 1]
```diff
@@ -1234,7 +1230,9 @@ def test_separate_users(self):
         assert_equal([self.user_4._id], removed)
 
     def test_event_subs_same(self):
-        self.file_sub.email_transactional.extend([self.user_2, self.user_3, self.user_4])
+        self.file_sub.email_transactional.add(self.user_2)
```
Comment (cwisecarver) [inline tests/test_notifications.py]: [self.file_sub.email_transactional.add(self.user_2, self.user_3, self.user4)](http://stackoverflow.com/questions/4959499/how-to-add-multiple-objects-to-manytomany-relationship-at-once-in-django?answertab=votes#tab-top)

File: website/notifications/emails.py [hunk 2]
```diff
@@ -147,7 +147,7 @@ def check_node(node, event):
 
 def get_user_subscriptions(user, event):
     user_subscription = NotificationSubscription.load(utils.to_subscription_key(user._id, event))
-    return {key: getattr(user_subscription, key, []) for key in constants.NOTIFICATION_TYPES}
+    return {key: [u._id for u in getattr(user_subscription, key, []).all()] for key in constants.NOTIFICATION_TYPES}
```
Comment (cwisecarver) [inline website/notifications/emails.py]: `{key: getattr(user_subscription, key).all().values_list('guids___id', flat=True) for key in constants.NOTIFICATION_TYPES}`

If you try to call `.all()` on a list it's going to fail and values list will let you get specific fields even across relationships.

File: website/notifications/emails.py [hunk 3]
```diff
@@ -157,7 +157,7 @@ def get_node_lineage(node):
     lineage = [node._id]
 
     while node.parent_id:
-        node = website_models.Node.load(node.parent_id)
+        node = AbstractNode.load(node.parent_id)
```
Comment (cwisecarver) [inline website/notifications/emails.py]: Won't `node.parent_node` do the same thing?

File: website/notifications/emails.py [hunk 4]
```diff
@@ -167,7 +167,7 @@ def get_settings_url(uid, user):
     if uid == user._id:
         return web_url_for('user_notifications', _absolute=True)
 
-    node = website_models.Node.load(uid)
+    node = AbstractNode.load(uid)
```
Comment (cwisecarver) [inline website/notifications/emails.py]: Here too.

File: website/notifications/utils.py [hunk 5]
```diff
@@ -182,15 +186,15 @@ def get_configured_projects(user):
...
+            not isinstance(node, AbstractNode) or
             (subscription.none.filter(id=user.id).exists() and not node.parent_id) or
             node._id not in user.notifications_configured or
             node.is_collection
         ):
             continue
 
         while node.parent_id and not node.is_deleted:
-            node = Node.load(node.parent_id)
+            node = AbstractNode.load(node.parent_id)
```
Comment (cwisecarver) [inline website/notifications/utils.py]: Also, this `node = node.parent_node`

File: website/notifications/emails.py (hunk 4 above)
Comment (mfraezz) [inline website/notifications/emails.py]: Not here, no.

**Pull request merged**